x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/metrics/metrics.tsx

```diff
@@ -0,0 +1,471 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import { first, last } from 'lodash';
+import { i18n } from '@kbn/i18n';
+import {
+  Axis,
+  Chart,
+  niceTimeFormatter,
+  Position,
+  Settings,
+  TooltipValue,
+  PointerEvent,
+} from '@elastic/charts';
+import moment from 'moment';
+import { TabContent, TabProps } from '../shared';
+import { useSnapshot } from '../../../../hooks/use_snaphot';
+import { useWaffleOptionsContext } from '../../../../hooks/use_waffle_options';
+import { useSourceContext } from '../../../../../../../containers/source';
+import { findInventoryFields } from '../../../../../../../../common/inventory_models';
+import { convertKueryToElasticSearchQuery } from '../../../../../../../utils/kuery';
+import { SnapshotMetricType } from '../../../../../../../../common/inventory_models/types';
+import {
+  MetricsExplorerChartType,
+  MetricsExplorerOptionsMetric,
+} from '../../../../../metrics_explorer/hooks/use_metrics_explorer_options';
+import { Color } from '../../../../../../../../common/color_palette';
+import {
+  MetricsExplorerAggregation,
+  MetricsExplorerSeries,
+} from '../../../../../../../../common/http_api';
+import { MetricExplorerSeriesChart } from '../../../../../metrics_explorer/components/series_chart';
+import { createInventoryMetricFormatter } from '../../../../lib/create_inventory_metric_formatter';
+import { calculateDomain } from '../../../../../metrics_explorer/components/helpers/calculate_domain';
+import { getTimelineChartTheme } from '../../../../../metrics_explorer/components/helpers/get_chart_theme';
+import { useUiSetting } from '../../../../../../../../../../../src/plugins/kibana_react/public';
+import { euiStyled } from '../../../../../../../../../observability/public';
+import { ChartHeader } from './chart_header';
+import {
+  SYSTEM_METRIC_NAME,
+  USER_METRIC_NAME,
+  INBOUND_METRIC_NAME,
+  OUTBOUND_METRIC_NAME,
+  USED_MEMORY_METRIC_NAME,
+  FREE_MEMORY_METRIC_NAME,
+  CPU_CHART_TITLE,
+  LOAD_CHART_TITLE,
+  MEMORY_CHART_TITLE,
+  NETWORK_CHART_TITLE,
+} from './translations';
+import { TimeDropdown } from './time_dropdown';
+
+const TabComponent = (props: TabProps) => {
+  const cpuChartRef = useRef<Chart>(null);
+  const networkChartRef = useRef<Chart>(null);
+  const memoryChartRef = useRef<Chart>(null);
+  const loadChartRef = useRef<Chart>(null);
+  const [time, setTime] = useState(60 * 60 * 1000);
```

Comment:
Name this `ONE_MINUTE` somewhere in a `const` just so we don't have unlabeled numbers scattered around

Reply:
Sorry, `ONE_HOUR`. But I guess that reinforces my point haha